setup.py

```diff
@@ -35,5 +35,9 @@
         'Programming Language :: Python :: Implementation :: CPython',
         'Programming Language :: Python :: Implementation :: PyPy',
     ],
-    py_modules=["jdcal"]
+    # py_modules=["jdcal"],
+    packages=find_packages("src"),
+    package_dir={"": "src"},
```

Comment:
You should also mention the `package_data` here:

```suggestion
    package_data={"jdcal": ["py.typed"]},
    package_dir={"": "src"},
```